packages/zosfiles/__tests__/__system__/api/methods/mount/Mount.system.test.ts

```diff
@@ -0,0 +1,173 @@
+            user:defaultSystem.ssh.user,
+            password:defaultSystem.ssh.password,
+        });
+        volume = defaultSystem.datasets.vol;
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
+        fsname = getUniqueDatasetName(defaultSystem.zosmf.user);
+        const dirname = getUniqueDatasetName(defaultSystem.zosmf.user).split(".")[1];
+        mountPoint = "/tmp/" + dirname;
+
+        // Execute SSH to add mountpoint to temp directory, in case of delete failure
+        await Shell.executeSsh(thisSshSession, "mkdir " + mountPoint, jest.fn());
+
+        // Create a ZFS
+        let error;
+        let response;
+        zfsOptions.perms = perms;
+        zfsOptions.cylsPri = cylsPri;
+        zfsOptions.cylsSec = cylsSec;
+        zfsOptions.timeout = timeout;
+        zfsOptions.volumes = [volume];
+
+        try {
+            response = await Create.zfs(REAL_SESSION, fsname, zfsOptions);
+            Imperative.console.info("Response: " + inspect(response));
+        } catch (err) {
+            error = err;
+            Imperative.console.info("Error: " + inspect(error));
+        }
+    });
+
+    afterAll(async () => {
+        defaultSystem = testEnvironment.systemTestProperties;
+        const thisSshSession = new SshSession({
+            hostname:defaultSystem.ssh.host,
+            port:defaultSystem.ssh.port,
+            user:defaultSystem.ssh.user,
+            password:defaultSystem.ssh.password,
+        });
+
+        // Delete the ZFS
+        let response;
+        try {
+            response = await Delete.zfs(REAL_SESSION, fsname);
+        } catch (error) {
+            Imperative.console.info("Error: " + inspect(error));
+        }
+        await TestEnvironment.cleanUp(testEnvironment);
+
+        // Remove the mount point
+        await Shell.executeSsh(thisSshSession, "rmdir " + mountPoint, jest.fn());
+    });
+
+    it("should mount a FS to a mount point", async () => {
+        let response;
+        let error;
+        mountOptions["fs-type"] = fsType;
+        mountOptions.mode = mode;
+
+        try {
+            response = await Mount.fs(REAL_SESSION, fsname, mountPoint, mountOptions);
+            Imperative.console.info("Response: " + inspect(response));
+        } catch (e) {
+            error = e;
+            Imperative.console.info("Error: " + inspect(error));
+        }
+
+        expect(error).toBeUndefined();
+        expect(response).toBeTruthy();
+        expect(response.success).toBe(true);
+        expect(response.commandResponse).toContain(ZosFilesMessages.fsMountedSuccessfully.message);
+
+        try{
+            response = await List.zfs(REAL_SESSION, {fsname});
+            Imperative.console.info("Response: " + inspect(response));
+        } catch (e) {
+            error = e;
+            Imperative.console.info("Error: " + inspect(error));
+        }
+
+        expect(error).toBeUndefined();
+        expect(response).toBeTruthy();
```

Comment:
Just curious, why `toBeTruthy` vs `toBeDefined`?

Reply:
We were going off of some of the other tests for ours, and found quite a few tests that used `expect(response).toBeTruthy`.